appveyor.yml

```diff
@@ -22,8 +23,9 @@ install:
   - "powershell ci-helpers/appveyor/install-miniconda.ps1"
   - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
   - "activate test"
-  - "pip install --quiet -r requirements/test.txt"
-  - "pip install --quiet spiceypy --no-cache-di"
+  - cmd: conda install --yes hdf5
+  - "pip install -r requirements/test.txt"
+  - "pip install spiceypy --no-cache-di"
```

Comment:
I don't think this line is needed any more (spiceypy now installs fine without --no-cache-di)